abba/json/Makefile

```diff
@@ -0,0 +1,30 @@
+NAME = json
+
+SRC = 	src/main.cpp \
+		src/Json.cpp \
+		src/Parse.cpp
+
+INCLUDE = -I ./include
+OBJ = $(SRC:%.cpp=%.o)
+
+
+CFLAGS = -Wall -Werror -Wextra -std=c++11 #-g -fsanitize=address
+
+all: build
+
+build:
+	$(MAKE) -j4 $(NAME)
+
+$(NAME): $(OBJ)
```

Comment:
Since you already made a CMakeLists file you can just call the CMake commands here, no?